### PR TITLE
PT-2932: Push field selection dialogue flashes annoyingly when opened

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by AlexLapena** [comment]
+1

File: components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
  &lt;div class="checkbox_tree_container"&gt;
  &lt;h5&gt;&lt;label for=""&gt;&lt;input type="checkbox" name="" id="" value="" /&gt;&lt;span class="section-name-header field-no-user-select"&gt;$section.name&lt;/span&gt;&lt;/label&gt;&lt;/h5&gt;
- &lt;ul class="field-list"&gt;
+ &lt;ul class="field-list #if($velocityCount &gt; 1)v-collapsed#end"&gt;
```
Comment (sdumitriu) [inline components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml]: `velocityCount` is deprecated, use `$foreach.index > 0` instead.

**Comment by veronikaslc** [comment]
Can we reuse `.addClassName('loading-indicator-large')` to the container as we do for loading process in the beginning of the push dialogs chain?

**Comment by veronikaslc** [comment]
added spinner icon after next button

**Comment by marta-** [comment]
I've made a new commit to improve the look of the spinner, please re-review.

image

**Pull request merged**